2021-09-30 11:40:21,890 INFO    MainThread:33131 [wandb_setup.py:_flush():69] setting env: {}
2021-09-30 11:40:21,890 INFO    MainThread:33131 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-30 11:40:21,890 INFO    MainThread:33131 [wandb_init.py:_log_setup():342] Logging user logs to /Users/obara/nd0821-c2-build-model-workflow-exercises/lesson-2-data-exploration-and-preparation/exercises/exercise_6/starter/wandb/run-20210930_114021-2jnod8kz/logs/debug.log
2021-09-30 11:40:21,890 INFO    MainThread:33131 [wandb_init.py:_log_setup():343] Logging internal logs to /Users/obara/nd0821-c2-build-model-workflow-exercises/lesson-2-data-exploration-and-preparation/exercises/exercise_6/starter/wandb/run-20210930_114021-2jnod8kz/logs/debug-internal.log
2021-09-30 11:40:21,891 INFO    MainThread:33131 [wandb_init.py:init():375] calling init triggers
2021-09-30 11:40:21,891 INFO    MainThread:33131 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-09-30 11:40:21,891 INFO    MainThread:33131 [wandb_init.py:init():424] starting backend
2021-09-30 11:40:21,891 INFO    MainThread:33131 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2021-09-30 11:40:21,900 INFO    MainThread:33131 [backend.py:ensure_launched():135] starting backend process...
2021-09-30 11:40:21,907 INFO    MainThread:33131 [backend.py:ensure_launched():139] started backend process with pid: 33140
2021-09-30 11:40:21,909 INFO    MainThread:33131 [wandb_init.py:init():429] backend started and connected
2021-09-30 11:40:21,911 INFO    MainThread:33131 [wandb_init.py:init():477] updated telemetry
2021-09-30 11:40:21,911 INFO    MainThread:33131 [wandb_init.py:init():500] communicating current version
2021-09-30 11:40:22,289 INFO    MainThread:33131 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-09-30 11:40:22,289 INFO    MainThread:33131 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-30 11:40:22,516 INFO    MainThread:33131 [wandb_init.py:init():540] starting run threads in backend
2021-09-30 11:40:22,999 INFO    MainThread:33131 [wandb_run.py:_console_start():1601] atexit reg
2021-09-30 11:40:22,999 INFO    MainThread:33131 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.REDIRECT
2021-09-30 11:40:22,999 INFO    MainThread:33131 [wandb_run.py:_redirect():1480] Redirecting console.
2021-09-30 11:40:23,000 INFO    MainThread:33131 [wandb_run.py:_redirect():1536] Redirects installed.
2021-09-30 11:40:23,000 INFO    MainThread:33131 [wandb_init.py:init():565] run started, returning control to user process
2021-09-30 11:40:23,180 INFO    MainThread:33131 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 1
2021-09-30 11:40:23,180 INFO    MainThread:33131 [wandb_run.py:_restore():1543] restore
2021-09-30 11:40:25,539 INFO    MainThread:33131 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1241
  total_bytes: 1241
}

2021-09-30 11:40:26,659 INFO    MainThread:33131 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1241
  total_bytes: 1241
}

2021-09-30 11:40:26,874 INFO    MainThread:33131 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1241
  total_bytes: 1241
}

2021-09-30 11:40:27,567 INFO    MainThread:33131 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1241
  total_bytes: 7793
}

2021-09-30 11:40:27,674 INFO    MainThread:33131 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1241
  total_bytes: 7793
}

2021-09-30 11:40:27,782 INFO    MainThread:33131 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 1241
  total_bytes: 7793
}

2021-09-30 11:40:27,888 INFO    MainThread:33131 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7793
  total_bytes: 7793
}

2021-09-30 11:40:27,996 INFO    MainThread:33131 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7793
  total_bytes: 7793
}

2021-09-30 11:40:28,100 INFO    MainThread:33131 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7793
  total_bytes: 7793
}

2021-09-30 11:40:28,207 INFO    MainThread:33131 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7793
  total_bytes: 7793
}

2021-09-30 11:40:28,316 INFO    MainThread:33131 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7793
  total_bytes: 7793
}

2021-09-30 11:40:28,557 INFO    MainThread:33131 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7793
  total_bytes: 7793
}

2021-09-30 11:40:28,667 INFO    MainThread:33131 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7793
  total_bytes: 7793
}

2021-09-30 11:40:29,749 INFO    MainThread:33131 [wandb_run.py:_show_files():1915] logging synced files
